src/WebJobs.Extensions.DurableTask/EndToEndTraceHelper.cs

```diff
@@ -16,12 +16,14 @@ internal class EndToEndTraceHelper
         private static string slotName;
 
         private readonly ILogger logger;
+        private readonly DurableTaskExtension durableTaskExtension;
 
         private long sequenceNumber;
 
-        public EndToEndTraceHelper(JobHostConfiguration config, ILogger logger)
+        public EndToEndTraceHelper(JobHostConfiguration config, ILogger logger, DurableTaskExtension durableTaskExtension)
```

Comment:
Rather than passing a reference to the extension, can we simply pass the value of `LogReplayEvents`? I feel like that would make it easier to add unit testing for this class.

Reply:
Great! I will change it. Thank you!